csvcubed/csvcubed/readers/skoscodelistreader.py

```diff
@@ -52,10 +58,18 @@ def extract_code_list_concept_scheme_info(
             + f"Expected 1, found {len(variables_in_about_url)}"
         )
 
-    if "notation" not in variables_in_about_url:
-        raise ValueError(
-            "Unexpected variable found in aboutUrl template. Expected 'notation', "
-            + f"found '{variables_in_about_url.pop()}'"
+    variable_name_in_about_url = first(variables_in_about_url)
+    assert variable_name_in_about_url is not None
+
+    if variable_name_in_about_url != "notation":
+        _logger.debug(
+            'Replacing variable name "%s" in URI template with "notation" for consistency.',
+            variable_name_in_about_url,
+        )
+        about_url = re.sub(
+            "\\{(.?)" + re.escape(variable_name_in_about_url) + "\\}",
+            "{\\1notation}",
```

Comment:
Do we need to guarantee the uniqueness of the variable name? If so, we can use uuid() here.

Reply:
No, we don't want it to be unique - in fact we want it to be consistent. It gets replaced in the caller by a suitable value.